### PR TITLE
add more tables and data

### PR DESCRIPTION
**Pull request by Divinity-dev**
In this phase of the project, I have answered the following questions;
1 Who was the last animal seen by William Tatcher?
imgA
2 How many different animals did Stephanie Mendez see?
imgB
3 List all vets and their specialties, including vets with no specialties.
imgC
4 List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.
imgD
5 What animal has the most visits to vets?
imgE
6 Who was Maisy Smith's first visit?
imgF
7 Details for most recent visit: animal information, vet information, and date of visit.
imgG
8 How many visits were with a vet that did not specialize in that animal's species?
imgI
9 What specialty should Maisy Smith consider getting? Look for the species she gets the most.
imgJ

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -46,4 +46,37 @@ update animals set owner_id = '1' where name = 'Agumon'
...
+('Maisy Smith', 26, '2019-01-17'),
+('Stephanie Mendez', 64, '1981-05-04'),
+('Jack Harkness', 38, '2008-06-08');
+--insert data into specialization table
+INSERT INTO specialization(species_id, vets_id)VALUES
+(1, 1),
+(2, 3),
+(1, 3),
+(2, 4);
+--insert data into visits table
```
Comment (BenMukebo) [inline data.sql:62]: * [OPTIONAL] This insertion could have been done dynamically where you do not know the `vet_id` and `specie_id`.

```sql
 INSERT INTO specializations (species_id, vet_id)
 VALUES ((SELECT id FROM vets WHERE name = 'William Tatcher'), (SELECT id FROM species WHERE name = 'Pokemon'));

 INSERT INTO specializations (species_id, vet_id)
 VALUES ((SELECT id FROM vets WHERE name = 'Stephanie Mendez'), (SELECT id FROM species WHERE name = 'Pokemon'));

 INSERT INTO specializations (species_id, vet_id)
 VALUES ((SELECT id FROM vets WHERE name = 'Stephanie Mendez'), (SELECT id FROM species WHERE name = 'Digimon'));

 INSERT INTO specializations (species_id, vet_id)
 VALUES ((SELECT id FROM vets WHERE name = 'Jack Harkness'), (SELECT id FROM species WHERE name = 'Digimon'));
```

File: data.sql [hunk 2]
```diff
@@ -46,4 +46,37 @@ update animals set owner_id = '1' where name = 'Agumon'
...
+INSERT INTO visits(animal_id, vets_id, date_of_visit) VALUES(1, 1, '2020-05-24');
+INSERT INTO visits(animal_id, vets_id, date_of_visit) VALUES(1, 3, '2020-07-22');
+INSERT INTO visits(animal_id, vets_id, date_of_visit) VALUES(2, 4, '2021-02-02');
```
Comment (BenMukebo) [inline data.sql:65]: - [OPTIONAL] The same here: 

```sql
 INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Agumon'), (SELECT id FROM vets WHERE name = 'William Tatcher'), '2020-05-24');
 ....
```

**Review by BenMukebo — APPROVED** [review]
### Hi @Divinity-dev,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻
- Gitflow is correctly used. 🥇 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

- [ ] Please remove all those placeholder comments in your README file, there is only an explanation on how you can customize your README file based on your own information 👍🏻 



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Divinity-dev** [comment]
Thank you 

Sent from Yahoo Mail on Android 
 
  On Fri, Apr 7, 2023 at 12:13 PM, Kasongo Mukebo ***@***.***> wrote:   


@BenMukebo approved this pull request.

Hi @Divinity-dev, 👋
 
STATUS: Approved 🎉
 
Great work on making the changes requested by a previous reviewer 🥇
Your project is complete! There is nothing else to say other than... it's time to merge it
Congratulations! 🎉 🎉 🎉
 


To Highlight 🥇
    
   - Your Pull Request has a proper title and description. ✔️

Optional suggestions

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding! 👏 👏 👏
 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

As described in the Code reviews limits policy you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form.

In data.sql:
> +INSERT INTO specialization(species_id, vets_id)VALUES
+(1, 1),
+(2, 3),
+(1, 3),
+(2, 4);
+--insert data into visits table
   
   - [OPTIONAL] This insertion could have been done dynamically where you do not know the vet_id and specie_id.
 INSERT INTO specializations (species_id, vet_id)
 VALUES ((SELECT id FROM vets WHERE name = 'William Tatcher'), (SELECT id FROM species WHERE name = 'Pokemon'));

 INSERT INTO specializations (species_id, vet_id)
 VALUES ((SELECT id FROM vets WHERE name = 'Stephanie Mendez'), (SELECT id FROM species WHERE name = 'Pokemon'));

 INSERT INTO specializations (species_id, vet_id)
 VALUES ((SELECT id FROM vets WHERE name = 'Stephanie Mendez'), (SELECT id FROM species WHERE name = 'Digimon'));

 INSERT INTO specializations (species_id, vet_id)
 VALUES ((SELECT id FROM vets WHERE name = 'Jack Harkness'), (SELECT id FROM species WHERE name = 'Digimon'));
In data.sql:
> +INSERT INTO visits(animal_id, vets_id, date_of_visit) VALUES(1, 1, '2020-05-24');
+INSERT INTO visits(animal_id, vets_id, date_of_visit) VALUES(1, 3, '2020-07-22');
+INSERT INTO visits(animal_id, vets_id, date_of_visit) VALUES(2, 4, '2021-02-02');
   
   - [OPTIONAL] The same here:
 INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Agumon'), (SELECT id FROM vets WHERE name = 'William Tatcher'), '2020-05-24');
 ....

**Pull request merged**